compiler/psi/src/org/jetbrains/kotlin/psi/KtNamedFunction.java

```diff
@@ -182,6 +182,37 @@ public KtTypeReference getReceiverTypeReference() {
         return getReceiverTypeRefByTree();
     }
 
+    @Nullable
+    @Override
+    public KtTypeReference getAdditionalReceiverTypeReference() {
```

Comment:
Almost same with `getReceiverTypeReference` above. Not sure if it is better to put common part to separate private method.